buildSrc/src/main/java/co/anitrend/buildSrc/plugins/strategy/DependencyStrategy.kt

```diff
@@ -37,6 +37,7 @@ internal class DependencyStrategy(private val project: Project) {
 
         test(project.libs.jetbrains.kotlin.test)
         test(project.libs.mockk)
+        test(project.libs.junit)
```

Comment:
Re-adding **junit** back, while not really used anymore I will create a seperate PR to remove it to keep track of changes in an atomic manner